tasks/enable.yml

```diff
@@ -0,0 +1,5 @@
+---
+- name: Start vertica
```

Comment:
This just enables, it doesn't actually start, right? Then we should change the task name to reflect that